CRITICAL - 2024-01-14 12:28:17 --> App\Controllers\Home::index(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
in APPPATH\Controllers\Home.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:28:18 --> App\Controllers\Home::index(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
in APPPATH\Controllers\Home.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:34:07 --> App\Controllers\Home::index(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
in APPPATH\Controllers\Home.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:34:07 --> App\Controllers\Home::index(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
in APPPATH\Controllers\Home.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:34:25 --> App\Controllers\Home::index(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
in APPPATH\Controllers\Home.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:34:29 --> App\Controllers\Home::index(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
in APPPATH\Controllers\Home.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 12:35:26 --> App\Controllers\Home::index(): Return value must be of type string, CodeIgniter\HTTP\RedirectResponse returned
in APPPATH\Controllers\Home.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 15:18:12 --> Class "App\Controllers\Time" not found
in APPPATH\Controllers\Admin.php on line 302.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 15:20:23 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 768.
 1 SYSTEMPATH\BaseModel.php(768): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([], false)
 3 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 4 APPPATH\Controllers\Admin.php(303): CodeIgniter\BaseModel->save([...])
 5 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 6 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 7 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 15:54:46 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 768.
 1 SYSTEMPATH\BaseModel.php(768): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH\Controllers\Admin.php(303): CodeIgniter\Model->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 15:58:04 --> mysqli_sql_exception: Unknown column 'role' in 'field list' in C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\new xampp\htdocs\gst\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\new xampp\htdocs\gst\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\new xampp\htdocs\gst\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\new xampp\htdocs\gst\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\new xampp\htdocs\gst\app\Controllers\Admin.php(303): CodeIgniter\Model->insert(Array)
#8 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
#9 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\new xampp\htdocs\gst\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-01-14 15:58:04 --> Unknown column 'role' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`role`) VALUES (:role:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin.php(303): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 15:58:20 --> mysqli_sql_exception: Unknown column 'role' in 'field list' in C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\new xampp\htdocs\gst\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\new xampp\htdocs\gst\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\new xampp\htdocs\gst\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\new xampp\htdocs\gst\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\new xampp\htdocs\gst\app\Controllers\Admin.php(303): CodeIgniter\Model->insert(Array)
#8 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
#9 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\new xampp\htdocs\gst\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-01-14 15:58:20 --> Unknown column 'role' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`role`) VALUES (:role:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin.php(303): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 15:59:05 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 768.
 1 SYSTEMPATH\BaseModel.php(768): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH\Controllers\Admin.php(303): CodeIgniter\Model->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 16:00:05 --> mysqli_sql_exception: Unknown column 'otp_expires_at' in 'field list' in C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\new xampp\htdocs\gst\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\new xampp\htdocs\gst\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\new xampp\htdocs\gst\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\new xampp\htdocs\gst\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\new xampp\htdocs\gst\app\Controllers\Admin.php(303): CodeIgniter\Model->insert(Array)
#8 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
#9 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\new xampp\htdocs\gst\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-01-14 16:00:05 --> Unknown column 'otp_expires_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`otp_expires_at`) VALUES (:otp_expires_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin.php(303): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 16:00:39 --> mysqli_sql_exception: Unknown column 'otp_expires_at' in 'field list' in C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\new xampp\htdocs\gst\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\new xampp\htdocs\gst\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\new xampp\htdocs\gst\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\new xampp\htdocs\gst\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\new xampp\htdocs\gst\app\Controllers\Admin.php(303): CodeIgniter\Model->insert(Array)
#8 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
#9 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\new xampp\htdocs\gst\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-01-14 16:00:40 --> Unknown column 'otp_expires_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`otp_expires_at`) VALUES (:otp_expires_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin.php(303): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 16:09:40 --> mysqli_sql_exception: Unknown column 'otp_expires_at' in 'field list' in C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\new xampp\htdocs\gst\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\new xampp\htdocs\gst\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\new xampp\htdocs\gst\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\new xampp\htdocs\gst\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\new xampp\htdocs\gst\app\Controllers\Admin.php(303): CodeIgniter\Model->insert(Array)
#8 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
#9 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\new xampp\htdocs\gst\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-01-14 16:09:40 --> Unknown column 'otp_expires_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`otp_expires_at`) VALUES (:otp_expires_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin.php(303): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 16:09:44 --> mysqli_sql_exception: Unknown column 'otp_expires_at' in 'field list' in C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\new xampp\htdocs\gst\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\new xampp\htdocs\gst\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\new xampp\htdocs\gst\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\new xampp\htdocs\gst\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\new xampp\htdocs\gst\app\Controllers\Admin.php(303): CodeIgniter\Model->insert(Array)
#8 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
#9 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\new xampp\htdocs\gst\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-01-14 16:09:44 --> Unknown column 'otp_expires_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`otp_expires_at`) VALUES (:otp_expires_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin.php(303): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 16:10:39 --> mysqli_sql_exception: Unknown column 'otp_expires_at' in 'field list' in C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\new xampp\htdocs\gst\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\new xampp\htdocs\gst\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\new xampp\htdocs\gst\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\new xampp\htdocs\gst\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\new xampp\htdocs\gst\app\Controllers\Admin.php(303): CodeIgniter\Model->insert(Array)
#8 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
#9 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\new xampp\htdocs\gst\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-01-14 16:10:39 --> Unknown column 'otp_expires_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`otp_expires_at`) VALUES (:otp_expires_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin.php(303): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 16:11:06 --> mysqli_sql_exception: Unknown column 'otp_expires_at' in 'field list' in C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\new xampp\htdocs\gst\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\new xampp\htdocs\gst\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\new xampp\htdocs\gst\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\new xampp\htdocs\gst\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\new xampp\htdocs\gst\app\Controllers\Admin.php(303): CodeIgniter\Model->insert(Array)
#8 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
#9 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\new xampp\htdocs\gst\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-01-14 16:11:06 --> Unknown column 'otp_expires_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`otp_expires_at`) VALUES (:otp_expires_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin.php(303): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-14 16:11:24 --> mysqli_sql_exception: Unknown column 'otp_expires_at' in 'field list' in C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\new xampp\htdocs\gst\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\new xampp\htdocs\gst\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\new xampp\htdocs\gst\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\new xampp\htdocs\gst\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\new xampp\htdocs\gst\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\new xampp\htdocs\gst\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\new xampp\htdocs\gst\app\Controllers\Admin.php(303): CodeIgniter\Model->insert(Array)
#8 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
#9 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 C:\xampp\new xampp\htdocs\gst\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\new xampp\htdocs\gst\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-01-14 16:11:24 --> Unknown column 'otp_expires_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`otp_expires_at`) VALUES (:otp_expires_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin.php(303): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 16:12:45 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 768.
 1 SYSTEMPATH\BaseModel.php(768): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH\Controllers\Admin.php(302): CodeIgniter\Model->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 16:14:26 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 768.
 1 SYSTEMPATH\BaseModel.php(768): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH\Controllers\Admin.php(302): CodeIgniter\Model->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Admin->sendOTP()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 16:21:21 --> Invalid file: "login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('login', [], true)
 3 APPPATH\Controllers\Home.php(16): view('login')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-14 16:21:21 --> Invalid file: "login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('login', [], true)
 3 APPPATH\Controllers\Home.php(16): view('login')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
